api/src/main/java/org/openmrs/module/registrationcore/api/mpi/openempi/PatientIdentifierMapper.java

```diff
@@ -21,23 +21,34 @@
     @Qualifier("registrationcore.mpiProperties")
     private MpiProperties mpiProperties;
 
-    public Integer getMappedLocalIdentifierTypeId(Integer mpiIdentifierTypeId) {
+    public String getMappedLocalIdentifierTypeId(String mpiIdentifierTypeId) {
         validateInit();
         for (IdentifierMapPair pair : MAPPED_ID) {
-            if (pair.mpiIdentifierId.equals(mpiIdentifierTypeId))
+            if (pair.mpiIdentifierId.equals(mpiIdentifierTypeId)) {
                 return pair.localIdentifierId;
+            }
         }
-        throw new IllegalArgumentException("There is no mapped local identifier type " +
-                "for mpi identifier type id: " + mpiIdentifierTypeId);
+        return null;
     }
 
-    public Integer getMappedMpiIdentifierTypeId(Integer localIdentifierTypeId) {
+    public String getMappedMpiIdentifierTypeId(String localIdentifierTypeId) {
         validateInit();
         for (IdentifierMapPair pair : MAPPED_ID) {
-            if (pair.localIdentifierId.equals(localIdentifierTypeId))
+            if (pair.localIdentifierId.equals(localIdentifierTypeId)) {
                 return pair.mpiIdentifierId;
+            }
         }
-        throw new IllegalArgumentException("There is no mapped mpi identifier type " +
+        return null;
+    }
+
+    public String getMappedMpiUniversalIdType(String localIdentifierTypeId) {
+        validateInit();
+        for (IdentifierMapPair pair : MAPPED_ID) {
+            if (pair.localIdentifierId.equals(localIdentifierTypeId)) {
+                return pair.universalIdType;
+            }
+        }
+        throw new IllegalArgumentException("There is no mapped universal id type " +
```

Comment:
any reason we are throwing an exception only in this method? we should be consistent